Altis_Life.Altis/dialog/function/fn_gangBankResponse.sqf

```diff
@@ -0,0 +1,16 @@
+#include "..\..\script_macros.hpp"
+/*
+    File: fn_gankBankResponse.sqf
+    Author: DomT602
+    Description:
+    Receives response from the server.
+*/
+params [
+    ["_value",-1,[0]]
+];
+if (!isRemoteExecuted || remoteExecutedOwner != 2) exitWith {};
```

Comment:
This isn't 2 for HC. Also last time I checked if the RE was from the HC it was simply broken. Might be worth checking before updating